bitbots_vision/bitbots_vision/dynamic_color_lookup_table.py

```diff
@@ -2,7 +2,7 @@
 
 import cv2
 import yaml
-import rospy
+import rclpy
```

Comment:
Do we want to migrate this as well or are we ignoring this as we will probably move to YOEO?

Reply:
I don't think we should put much effort into the dynamic colorspace.